orca-igor/src/main/groovy/com/netflix/spinnaker/orca/igor/pipeline/JenkinsStage.groovy

```diff
@@ -17,72 +17,10 @@
 package com.netflix.spinnaker.orca.igor.pipeline
 
 import groovy.transform.CompileStatic
-import groovy.util.logging.Slf4j
-import com.netflix.spinnaker.orca.CancellableStage
-import com.netflix.spinnaker.orca.ExecutionStatus
-import com.netflix.spinnaker.orca.batch.RestartableStage
-import com.netflix.spinnaker.orca.igor.tasks.MonitorJenkinsJobTask
-import com.netflix.spinnaker.orca.igor.tasks.MonitorQueuedJenkinsJobTask
-import com.netflix.spinnaker.orca.igor.tasks.StartJenkinsJobTask
-import com.netflix.spinnaker.orca.igor.tasks.StopJenkinsJobTask
-import com.netflix.spinnaker.orca.pipeline.StageDefinitionBuilder
-import com.netflix.spinnaker.orca.pipeline.TaskNode
-import com.netflix.spinnaker.orca.pipeline.model.Execution
-import com.netflix.spinnaker.orca.pipeline.model.Stage
-import com.netflix.spinnaker.orca.pipeline.model.Task
-import com.netflix.spinnaker.orca.pipeline.persistence.ExecutionRepository
-import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.stereotype.Component
 
-@Slf4j
 @Component
 @CompileStatic
-class JenkinsStage implements StageDefinitionBuilder, RestartableStage, CancellableStage {
-  @Autowired StopJenkinsJobTask stopJenkinsJobTask
+class JenkinsStage extends BuildStage {
```

Comment:
Also not sure we have a good story renaming tasks. IIRC changing these task names makes trying to load a previous pipeline / execution incompatible with the previous release. Not really comfortable doing this massive renaming of tasks for the sake of supporting the travis stage. I would roll back these changes and just have a TravisStage extends JenkinsStage